lambda/skill/i18n/language_model_it.py

```diff
@@ -0,0 +1,98 @@
+from datetime import tzinfo
+from skill.i18n.language_model_abc import LanguageModelABC
+
+
+class LanguageModelIT(LanguageModelABC):
+    def __init__(self, timezone: tzinfo):
+        self.timezone = timezone
+        self.SKILL_NAME = 'Telegram Connect'
+        self.SKILL_NAME_SPOKEN_IT = "<lang xml:lang='it-IT'>Telegram Connect</lang>".format(self.SKILL_NAME)
```

Comment:
Please change that back to:
``
self.SKILL_NAME_SPOKEN_EN = "<lang xml:lang='en-US'>Telegram Connect</lang>".format(self.SKILL_NAME)
``
This specifies that Alexa will say 'Telegram Connect' in an English accent. If you want that Alexa says it in an Italian way you can just do something like:
``
self.SKILL_NAME_SPOKEN_EN = self.SKILL_NAME
``
However, in German she says it in a weird way so I forced her to pronounce it in English, since the skill name are English words. Addtitionally, `self.SKILL_NAME_SPOKEN_EN` is used in different places that is way it wouldn't work if we change the name of the variable.

If you want, you can also update the testcase inside lambda/skill_test/test_language_mode.py. This test checks whether all the defined `LanguageModel`s have the same amount of attributes and the names of attributes must all be identical.